packages/manager/modules/billing/package.json

```diff
@@ -28,6 +28,7 @@
     "angular": "^1.7.5",
     "angular-translate": "^2.18.1",
     "bootstrap": "~3.3.7",
-    "bootstrap4": "twbs/bootstrap#v4.0.0"
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
```

Comment:
bootstrap 4 required? if so is it possible to remove bootstrap 3?

Reply:
bootstrap4 required for utilities, bootstrap 3 for the grid (still bootstrap3 in Dedicated app)